.structure-config

```diff
@@ -1 +1 @@
-INFRASTRUCTURE_FILES='.structure-config .packit.yml .pep8speaks.yml .codecov.yml .coveragerc .github/ dockerfile/ scripts/testing/ scripts/githooks/'
+INFRASTRUCTURE_FILES='.structure-config .packit.yml .readthedocs.yaml .pep8speaks.yml .codecov.yml .coveragerc .github/ dockerfile/ scripts/testing/ scripts/githooks/'
```

Comment:
To resolve infra check fail, please add here also `docs/requirements.txt` file.

Reply:
Fixed.